feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts

```diff
@@ -0,0 +1,13 @@
+import { OccConfig } from '@spartacus/core';
+
+export const defaultOccCheckoutB2BConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        paymentTypes: 'paymenttypes',
+        setCartCostCenter: 'users/${userId}/carts/${cartId}/costcenter',
```

Comment:
remind me why we don't get everything (additional properties) from before

it's because of the super duper checkout query right 🦸

Reply:
Yes. And no need to transfer extra bytes, as it's not needed :)